### ISSUE TITLE
Seasons won't be bought

### ISSUE THREAD

**Issue by Mtarnuhal** [Mtarnuhal/FrozenCookies#51]
Seasonal Biscuits aren't in the recommandation list any more and won't be bought anymore.

**Comment by adavidw** [comment]
Were they in the recommendation list before? The way I remember it is that they wouldn't show in the list, but they were calculated like a chain, so that if the halloween cookies were next most efficient to buy, it would then change seasons to halloween to get them.

I know there's code in there to not just change seasons willy-nilly, but to wait some period of time after last purchasing a season before purchasing another one. I wonder if there's any problem caused by the fact that it's currently halloween time, so the game starts in halloween season. Maybe it's not buying more right away because it hasn't been in Halloween long enough after a restart?

**Comment by Mtarnuhal** [comment]
Festive biscuit(Christmas season) was on the recommendation list.
![image](https://user-images.githubusercontent.com/2243505/97670403-6f4a1d00-1a86-11eb-94c1-3f4bcf76dd8c.png)

I've left it running for like a day after ascension to make sure it wasn't just waiting to get the cookies to buy it.
It didn't buy any seasons all upgrades from seasons were left in the recommendation list.
After manual buying a season it seems to switch to a different season.
Not sure if it is because we are in the Halloween season.

**Comment by adavidw** [comment]
Everything's behaving normally now that it's not Halloween by default. We'll have to look closer to determine why it wouldn't be working right when you start with a season that wasn't purchased.

**Comment by Mtarnuhal** [comment]
Yeah, might be worth checking why it's not continuing if it starts with a natural season.
